content/posts/manufacturers-are-also-just-middlemen.md

```diff
@@ -1,10 +1,9 @@
 ---
-layout: post
-title: Manufacturers are also just Middle Men
-tags:
- - digitalmanufacturing
+date: "2012-05-06T00:00:00Z"
```

Comment:
Since you are dropping the time from the datetime anyway (and no problem with that), can you just write all the date fields as `"yyyy-mm-dd"` instead of `"yyyy-mm-ddT00:00:00Z"` please?

Reply:
ok .